shortstories/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+class ShortStory(models.Model):
+     status=(
+        ('dr', 'Draft'),
+        ('p' , 'Published'),
+        ('b' , 'banned'),
+        ('de', 'deleted'),
+    )
+    author =models.Foreignkey(user,on_delete=models.CASCADE)
+    title = models.CharField(max_length=500)
+    body = models.CharField(min_length=3500,Choices = status)
```

Comment:
This is not correct, you're storing the shortstory body of text inside `body` you need another field to store the status in it.

Reply:
I did not get from the issue that I need to have another field to store status, what should be the name of that field?

Reply:
make it status and change the tuple above to PUBLICATION_STATUS, all caps

Reply:
done